mixer/adapter/stackdriver/log/log.go

```diff
@@ -201,6 +194,13 @@ func (h *handler) HandleLogEntry(_ context.Context, values []*logentry.Instance)
 		}
 		linfo.log(e)
 	}
+
+	for name, linfo := range h.info {
```

Comment:
this will flush all logs (not just the ones for which updates were received), correct?

Reply:
Correct. I didn't feel like adding tracking logic here, since there typically will be a very low number of logs (like 1 or 2). Flush is cheap if there's nothing there.